todo_app/data/view_model.py

```diff
@@ -1,6 +1,42 @@
+from todo_app import item
+from todo_app.data.item import Item
+
+
 class ViewModel:
-    def __init__(self, items: list[Items]):
+    def __init__(self, items: list[Item]):
         self._items = items
- 
+
+    @property
     def items(self):
-        return self._items
\ No newline at end of file
+        return self._items
+    
+    @property
+    def todo_items(self):
+        output =[]
+
+        for item in self._items:
+            if item.status == "To Do":
+                output.append(item)
+
+        return output
```

Comment:
Nice solution 👍 Not a recommendation necessarily, but I always think nice to compare how we can solve this more concisely (but debatably less-readably) with a list-comprehension:
```
        return [item for item in self._items if item.status == "To Do"]
```